.github/workflows/ci.yml

```diff
@@ -127,7 +127,7 @@ jobs:
                         libespeak-dev libespeak-ng-dev
                         libttspico-dev
                         libao-dev libasound2-dev libaudio-dev libpulse-dev libxau-dev libsndfile1-dev
-                        libdotconf-dev libglib2.0-dev libltdl-dev- systemd
+                        libdotconf-dev libglib2.0-dev libltdl-dev- libsystemd-dev libsystemd0 systemd
```

Comment:
You don't need to introduce `libsystemd0`, that'll come as a dependency from `libsystemd-dev` (and whenever the abi changes 0 would get bumped, which would require a change here, while `libsystemd-dev` alone will be fine)